k8s/pkg/apis/longhorn/v1beta2/volume.go

```diff
@@ -67,6 +67,15 @@ const (
 	ReplicaAutoBalanceBestEffort  = ReplicaAutoBalance("best-effort")
 )
 
+// +kubebuilder:validation:Enum="";disabled;enabled
+type UnmapMarkSnapChainRemoved string
+
+const (
+	UnmapMarkSnapChainRemovedEmpty    = UnmapMarkSnapChainRemoved("")
```

Comment:
Should we need the empty value? How about setting the empty value to `UnmapMarkSnapChainRemovedDisabled` during upgrade?

Reply:
This will be updated `UnmapMarkSnapChainRemoved("ignored")`.